packages/react-components/react-spinbutton/src/stories/SpinButton/SpinButtonAccessibility.md

```diff
@@ -0,0 +1,90 @@
+## Component features and behavior
+
+This section documents accessibility-related behavoirs of spin buttons.
+
+### Keyboarding
+
+#### Tab Order
+1. Value field
+
+#### States
+1. Committed (Focused and value committed)
+2. Editing (Focused and editing)
+
+#### Keyboard State Diagram
+
+| Starting state | Transition | Resulting state |
+| ---------- | ------------ | ------- |
+| Content before spin button | Tab | Committed |
+| Committed  | Tab | Content after spin button (Value committed) |
+| Editing  | Tab | Content after spin button (Value committed) |
+| Editing (Autofill option provided)  | Tab | Content after spin button (Value committed) |
+| Editing  | Enter | Committed |
+| Committed  | Any Edit Key (that results in a change) | Editing |
+| Editing  | Any Edit Key | Editing |
+| Content after spin button (Value committed) | Shift + Tab | Committed |
+
+##### Edit keys
+| Edit Key | Result |
+| ---------- | ------------ |
+| Home | First item in defined range |
+| End | Last item in defined range |
+| Up arrow | One increment higher |
+| Down arrow | One increment lower |
+| Page up | Several increments higher |
+| Page down | Several increments lower |
+| Typing valid value | Valid value |
+| Begin typing valid value | Autofill option provided |
+
+### High contrast mode
+
+Be careful about visibility of the spin button arrows in high contrast mode.
+
+### Motion and animation
+
+Animations occur when the value changes to give the appearance of spinning. This needs to respond to user settings to turn off animations.
+
+Placeholder: Mouse press speed of value changes visual indicators
+
+### Semantics
+
+## Known issues
+
+Placeholder: If there are any current known bugs with browsers or AT that impact this component, list them here.
+
+Mouse press quick changes to value are not announced to screen readers.
```

Comment:
This can go in a different PR, but I'm dropping this here for reference:

- NVDA doen't read the value of spinbuttons in Chromium: https://github.com/nvaccess/nvda/issues/13195
- Narrator defaults to reading min/max as 0 when they're (intentionally) undefined: https://microsoft.visualstudio.com/Edge/_workitems/edit/39070743